apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V91__add_clm_grp_index.sql

```diff
@@ -0,0 +1,28 @@
+-- ADD CLM_GRP_ID INDEX
+--
+-- This flyway script is intended to add an index on clm_grp_id for a handful
+-- of tables that need their clm_grp_ids cleaned up for certain synthetic data.
+-- This index is temporary and will be removed in a subsequent script.
+
+-- This is done in psql only since this index is only required for a one-time
+-- cleanup script against the actual database.
+
+-- =====================
+-- carrier_claims
+-- =====================
+${logic.psql-only} create index if not exists clm_grp_id_index on public.carrier_claims(clm_grp_id)
```

Comment:
we'll need to crete the indexs 'concurrently'...otherwise we'll lock the table while it build the index.

create index concurrently if not exists clm_grp_id_index on public.carrier_claims(clm_grp_id)

ditto for the others.

Reply:
Thanks cmac! New to this so appreciate that. Remember now you said that earlier but slipped my mind.

Reply:
Added concurrently!